src/mod/applications/mod_dptools/mod_dptools.c

```diff
@@ -934,6 +934,8 @@ SWITCH_STANDARD_APP(eavesdrop_function)
 		const char *bridge_bleg = switch_channel_get_variable(channel, "eavesdrop_bridge_bleg");
 		const char *whisper_aleg = switch_channel_get_variable(channel, "eavesdrop_whisper_aleg");
 		const char *whisper_bleg = switch_channel_get_variable(channel, "eavesdrop_whisper_bleg");
+		const char *bug_top = switch_channel_get_variable(channel, "eavesdrop_bug_top");
+		const char *bug_bottom = switch_channel_get_variable(channel, "eavesdrop_bug_bottom");
```

Comment:
are these mutually exclusive? maybe to add a check that both cannot be set at the same time, if they are exclusive

Reply:
ok